src/js/components/Video/Video.js

```diff
@@ -7,8 +7,8 @@ import React, {
   useRef,
   useState,
 } from 'react';
-import { ThemeContext } from 'styled-components';
 
+import { ThemeContext } from 'styled-components';
```

Comment:
'styled-components' is an external library so it should be on the upper import section. Just revert to how it was.